src/scss/_bases.scss

```diff
@@ -54,7 +60,7 @@ article {
 }
 
 body {
-    color: #73879C;
+    color: $default-light-blue;
     background: #2A3F54;
```

Comment:
Yeah, once we have the structure on point, we can start looking at the colors. We have to figure out if they are unique colors or just variations of the default colors using lighten(), darken() or similar.